products/ASC.Files/Client/src/pages/Home/Section/Body/RowsView/FilesRowContent.js

```diff
@@ -64,22 +87,19 @@ const FilesRowContent = ({
   sectionWidth,
   titleWithoutExt,
   updatedDate,
-  fileOwner,
   linkStyles,
   badgesComponent,
-  isAdmin,
 }) => {
   const {
     contentLength,
     fileExst,
     filesCount,
     foldersCount,
     providerKey,
-    access,
     title,
   } = item;
 
-  const withAccess = isAdmin || access === 0;
+  updatedDate = moment(new Date(updatedDate)).format("MM/D/YYYY	hh:mm A");
```

Comment:
MomentJS is to heavy for this. If you need remove comma from date string just replase it inside  "getStatusByDate" and "getTableStatusByDate" finctions at withContent HOC`s.

Reply:
Removed momentJS from FilesRowContent component. Combined the getStatusByDate and getTableStatusByDate methods into the getStatusByDate method in the withContent HOC`s. Added parameters for the toLocaleString method in the getStatusByDate method. Replaced the comma from date string inside the getStatusByDate method.